IDEOorg/steps-bot
# #161240348 Update Scripts to Return Org Phone Number

### What does this PR do?
This updates the scripts to return the client's org's phone number in the `followup` and `ultimatedone` scripts.

### Description of tasks to be completed
- update api request to return org phone number
- update rive scripts to use org phone number
- update tests to pass

### How should this be manually tested?
* Clone this repo
* Checkout to a branch `ft-add-organisation-number-161240348`.
* Run `npm start` and `npm run start:live:dev`
* Setup Twilio webhook.
* Log in as a coach and create a client if not already present.
* Add a task to the client's work plan and mark the task as completed.
* Visit [twilio-gui](http://twilio-gui.herokuapp.com) and get a phone number.
* login to `twilio-gui`
* Initiate the conversation with the bot by sending `hi`.
* Fast forward the conversation by sending `ff`.
* Note that the message sent to the client now includes the coach's org's phone number.

### What are the relevant pivotal tracker stories?
[#161240348](https://www.pivotaltracker.com/story/show/161240348)

### Any Background Context
There's a [corresponding PR](https://github.com/IDEOorg/steps/pull/508) on the Steps repo that allows an admin to add their organisation's phone number when signing up for the first time. 

### Screenshots?
<img width="1072" alt="screenshot 2019-02-25 at 4 37 13 pm" src="https://user-images.githubusercontent.com/36507972/53404067-2e4a5d00-39b5-11e9-8fad-995fd8655416.png">
<img width="895" alt="screenshot 2019-02-25 at 4 37 24 pm" src="https://user-images.githubusercontent.com/36507972/53404069-2e4a5d00-39b5-11e9-9584-b5462ae65594.png">

### Questions
N/A
